domains/vikrant.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "diluteoxygen",
+        "email": "diluteoxygen@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "CNAME"
```

???

it should be something like
```suggestion
        "CNAME": "your website url"
```

---

Fixed it!